### PR TITLE
new: Missing /account/transfer endpoint

### PR DESCRIPTION
**Pull request by yoshizzle**
* Created object and endpoint data for /account/transfer

### CODE REVIEW

**Comment by yoshizzle** [comment]
I could **_REALLY_** use some help with the object descriptions!

**Comment by Dorthu** [comment]
> bug: Missing /account/transfer endpoint

Might as well be a `new:`

File: src/data/objects/transfer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Transfer
+prefix: transfer
+description: >
+    Transfer pool statistics
+schema:
+  billable:
+    type: Integer
+    value: 0
+    description: Amount of billable pool.
```
Comment (RobertDeRose) [inline src/data/objects/transfer.yaml]: This reads weird to me..   maybe "How much of the pool is billable" ?

Comment (RobertDeRose) [reply src/data/objects/transfer.yaml]: Or "The amount of the pool that is billable"

Comment (yoshizzle) [reply src/data/objects/transfer.yaml]: I added to your suggestion slightly. Thanks.

**Comment by stvnjacobs** [comment]
This will close out https://jira.linode.com/browse/BIT-124. If you could resolve when this gets merged, I'd appreciate it.

**Pull request merged**